common/TelemetryEvents/SourceControlIntegration/SourceControlIntegrationHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Linq;
+
+namespace DevHome.Common.TelemetryEvents.SourceControlIntegration;
+
+public static class SourceControlIntegrationHelper
+{
+    public static string GetSafeRootPath(string rootPath)
```

Comment:
I'm not a fan of the name.  This isn't returning the root path, but a part of the root path.  The last part.  maybe `GetLowestDirectoryFromRootPath`, or `GetNameOfRepositoryDirectory`?

Reply:
Also, I do not agree that the word "safe" is appropriate because no names are being obfuscated.